boilerplates/react-union-boilerplate-liferay-basic/react-union-support/src/main/java/eu/lundegaard/reactunion/support/jackson/ObjectMapperFactory.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) Lundegaard, s.r.o. 2018 - All Rights Reserved
+ * Proprietary and confidential. Unauthorized copying of this file, via any medium is strictly prohibited.
+ */
+package eu.lundegaard.reactunion.support.jackson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @author Roman Srom (roman.srom@lundegaard.eu)
+ */
+public class ObjectMapperFactory {
+
+    public static ObjectMapper createObjectMapper() {
```

Comment:
I would prefere singleton instance. Not creating new instance everytime. Keep it in static final variable. It should be thread safe until you didn't change mapper config.

Reply:
react-union-support and react-union-taglib were removed because they are in a separated repository. Bug caused by old code and master merge.